packages/syft-cli/setup.py

```diff
@@ -0,0 +1,34 @@
+# third party
+from setuptools import find_packages
+from setuptools import setup
+from version import __version__
+
+packages = [
+    "typer[all]==0.9.0",
+    "typing_extensions==4.6.3",
```

Comment:
Q: I noticed we are not using typing_extensions currently or do we plan to use it later ?

Reply:
Yes for using `Annotated` type hint - https://typer.tiangolo.com/tutorial/options/help/